README.md

```diff
@@ -97,23 +89,23 @@ devcontainer build --workspace-folder <path-to-my-project-with-devcontainer>
 ...
 ```
 
--   [`src`](src) - A collection of subfolders, each declaring a feature. Each subfolder contains at least a
+-   [`src`](src) - A collection of subfolders, each declaring a Feature. Each subfolder contains at least a
     `devcontainer-feature.json` and an `install.sh` script.
 -   [`test`](test) - Mirroring `src`, a folder-per-feature with at least a `test.sh` script. The
     [`devcontainer` CLI](https://github.com/devcontainers/cli) will execute
     [these tests in CI](https://github.com/devcontainers/features/blob/main/.github/workflows/test-all.yaml).
 
 ## Contributions
 
-### Creating your own collection of features
+### Creating your own collection of Features
 
-The [feature distribution specification](https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-features-distribution.md) outlines a pattern for community members and organizations to self-author features in repositories they control.
+The [Feature distribution specification](https://github.com/devcontainers/spec/blob/main/docs/specs/devcontainer-features-distribution.md) outlines a pattern for community members and organizations to self-author Features in repositories they control.
```

Comment:
Should we point to the website instead, once https://github.com/devcontainers/devcontainers.github.io/pull/62 is merged?

Reply:
Sounds good, I'll update to https://containers.dev/implementors/features-distribution/